frontend/src/components/stat-cards/_base-stat-card.scss

```diff
@@ -89,6 +43,13 @@
     }
 
     .history-toggle {
+      // override those pesky LMS styles
```

Comment:
I sympathize with you, but maybe drop 'pesky' and say '// override default LMS styls'?